WalletWasabi/BitcoinCore/IRPCClient.cs

```diff
@@ -1,16 +1,144 @@
 using NBitcoin;
+using NBitcoin.RPC;
+using System;
 using System.Threading.Tasks;
 
 namespace WalletWasabi.BitcoinCore
 {
 	public interface IRPCClient
 	{
 		Network Network { get; }
+		RPCCredentialString CredentialString { get; }
 
-		Task<uint256> GetBestBlockHashAsync();
+		public Task<uint256> GetBestBlockHashAsync()
+		{
+			throw new NotImplementedException();
+		}
 
-		Task<Block> GetBlockAsync(uint256 blockId);
+		public Task<Block> GetBlockAsync(uint256 blockId)
+		{
+			throw new NotImplementedException();
+		}
 
-		Task<BlockHeader> GetBlockHeaderAsync(uint256 blockHash);
+		public Task<Block> GetBlockAsync(uint blockHeight)
+		{
+			throw new NotImplementedException();
+		}
+
+		public Task<BlockHeader> GetBlockHeaderAsync(uint256 blockHash)
+		{
+			throw new NotImplementedException();
+		}
+
+		public Task<uint256[]> GenerateAsync(int blockCount)
+		{
+			throw new NotImplementedException();
+		}
+
+		public Task StopAsync()
+		{
+			throw new NotImplementedException();
+		}
+
+		public Task<BlockchainInfo> GetBlockchainInfoAsync()
+		{
+			throw new NotImplementedException();
+		}
+
+		public Task<PeerInfo[]> GetPeersInfoAsync()
+		{
+			throw new NotImplementedException();
+		}
+		public Task<TimeSpan> UptimeAsync()
+		{
+			throw new NotImplementedException();
+		}
+
+		public Task<uint256> SendRawTransactionAsync(Transaction transaction)
+		{
+			throw new NotImplementedException();
+		}
+
+		public Task<MempoolEntry> GetMempoolEntryAsync(uint256 txid, bool throwIfNotFound = true)
+		{
+			throw new NotImplementedException();
+		}
+
+		public Task<uint256[]> GetRawMempoolAsync()
+		{
+			throw new NotImplementedException();
+		}
+
+		public Task<MempoolAcceptResult> TestMempoolAcceptAsync(Transaction transaction, bool allowHighFees = false)
+		{
+			throw new NotImplementedException();
+		}
+
+		GetTxOutResponse GetTxOut(uint256 txid, int index, bool includeMempool = true)
```

Comment:
Add access modifier.
```suggestion
		public GetTxOutResponse GetTxOut(uint256 txid, int index, bool includeMempool = true)
```

Reply:
Why?

Reply:
Using explicit access modifiers makes the code more clear IMO.
Also to be consistent. All the other methods added in this PR have explicit access modifier.